backend/config/.env.example

```
@@ -64,19 +64,68 @@ ADMIN_PASSWORD=your-secure-password
+# uv run --with boto3,httpx,cryptography scripts/replay_raw_payloads.py --user-id <user UUID within s3 bucket> --target-user-id <your's OW instance user UUID>
+# If payloads were stored encrypted (DATA_ENCRYPTION_KEY set), export the same key so the script can decrypt them.
 OPEN_WEARABLES_API_KEY=secret-key-from-openwearables-settings-credentials
 RAW_PAYLOAD_S3_BUCKET=s3-bucket-name
 RAW_PAYLOAD_S3_ENDPOINT_URL=https://s3-compatible-endpoint.com
 RAW_PAYLOAD_S3_PREFIX=optional-prefix-if-used
 # also remember to add AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY which are listed in section below
 
-#--- AWS ---#
+#--- AWS / S3-compatible object storage ---#
 AWS_BUCKET_NAME=open-wearables
 AWS_ACCESS_KEY_ID=your-access-id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_ACCESS_KEY_ID key should go before the AWS_BUCKET_NAME key

(UnorderedKey)